File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/ScrubIpFilterTypes.js
goog.provide('API.Client.ScrubIpFilterTypes');

/**
 * Available scrub filter types for building firewall rules.
 * @record
 */
API.Client.ScrubIpFilterTypes = function() {}

/**
 * @type {!boolean}
 * @export
 */
API.Client.ScrubIpFilterTypes.prototype.success;

/**
 * @type {!Object<!string, API.Client.ScrubIpFilterTypes_filters_value>}
 * @export
 */
API.Client.ScrubIpFilterTypes.prototype.filters;

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/ScrubIpFilterTypes.scala
package io.swagger.server.model


/**
 * Available scrub filter types for building firewall rules.
 *
 * @param success 
 * @param filters 
 */
case class ScrubIpFilterTypes (
  success: Option[Boolean],
  filters: Option[Map[String, ScrubIpFilterTypes_filters]]
)

